app/api/route/message_api.rb

```diff
@@ -0,0 +1,31 @@
+module Route
+  class MessageApi < Base
+    paginate paginate_settings
+
+    params do
+      optional :thread, type: Integer, desc: 'ID of thread'
```

Comment:
I'd prefer if it was called `:thread_id`

Reply:
OK